addOns/retire/src/main/javahelp/org/zaproxy/addon/retire/resources/help/contents/retire.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<TITLE>
+Retire.js
+</TITLE>
+</HEAD>
+<BODY>
+<H1>Retire.js</H1>
+
+This add-on includes a passive scan rule which implements checks provided by <a href="https://retirejs.github.io/retire.js/">Retire.js</a> 
+in order to identify vulnerable or out-dated JavaScript packages.
+
```

Comment:
Add `br` or `p` (same before the note)? They are all shown in the same line/paragraph.